.gitlab-ci.yml

```diff
@@ -34,13 +36,20 @@ stages:
   - infra-prod
   - production
 
+default:
+  cache:
+    paths:
+      # $CI_PROJECT_DIR is /builds/senbox-org/snap-engine/ on ci
+      - "${CI_PROJECT_DIR}/testData"
+
 mvn-build:
+  before_script:
+    - aws s3 sync s3://%S3_BUCKET%/testData/SAR/unittests %TEST_DATA_DIR%/SAR/unittests %S3_ARGS%
```

Comment:
Should be 
`- aws s3 sync s3://$S3_BUCKET/testData/SAR/unittests $TEST_DATA_DIR/SAR/unittests $S3_ARGS`